### PR TITLE
Add code reuse FAQ to NG Module Guide

### PR DESCRIPTION
**Pull request by rjcorwin**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: Purely just documentation.
```

**What is the current behavior?** (You can also link to an open issue here)
Folks expect that there is standard tooling for sharing Angular Modules.

**What is the new behavior?**
There is not standard tooling for sharing Angular Modules.

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

Related issue: https://github.com/angular/angular-cli/issues/6510

Which is a continuation of this heated issue: https://github.com/angular/angular-cli/issues/1692

### CODE REVIEW

**Comment by rjsteinert** [comment]
I signed it!

**Comment by rjsteinert** [comment]
If PR https://github.com/angular/angular/pull/16486 is merged, then this PR will not be necessary.

File: aio/content/guide/ngmodule-faq.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ These FAQs assume that you have read the [NgModules](guide/ngmodule) page.
 
 {@a q-what-to-declare}
 
+## How do I reuse Angular Modules in other projects?
 
+Currently there is no official way to package Angular Modules for resuse. Follow [this issue](https://github.com/angular/angular-cli/issues/6510) for updates.
```
Comment (vicb) [inline aio/content/guide/ngmodule-faq.md]: re-use

**Comment by vicb** [comment]
@rjsteinert I'm closing this because we plan to improve the docs (and the CI is red and there are 2 commits)

Thanks for your feedback

**Comment by rjsteinert** [comment]
Got it. I see the CI feedback now...

```
INVALID COMMIT MSG: "Merge pull request #1 from angular/master"
 => ERROR: The commit message does not match the format of "<type>(<scope>): <subject> OR revert: type(<scope>): <subject>"
Please fix the failing commit messages before continuing...
Commit message guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines
```

And I'll rebase master in the future so there aren't merge commits.

**Pull request closed without merging**